WordPress/gutenberg
# Fix: Adds help props for description of Play Inline toggle

## What?
Fixes https://github.com/WordPress/gutenberg/issues/60867

## Why?
- Adding a brief description for the 'Play inline' setting which clarifies its purpose, especially for users unfamiliar with its functionality, primarily affecting mobile browser video playback.

## How?
- It uses the help props in the ToggleControl components to add a brief description for it.

## Testing Instructions
1. Add a Video block and add a video to it.
2. Observe the Settings panel contains a setting named 'Play inline'.

### Testing Instructions for Keyboard
- NIL

## Screenshots or screencast 
<img width="1118" alt="image" src="https://github.com/WordPress/gutenberg/assets/77401999/5fc8a4fb-f5f8-4381-9130-689ace2c3b8b">
